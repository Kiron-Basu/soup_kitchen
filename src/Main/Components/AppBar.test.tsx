import React from "react";
import { ShallowWrapper, shallow } from "enzyme";

import ButtonAppBar from "./AppBar";

describe("App AppBar tests", () => {
  test("Renders AppBar correctly", () => {
    const component: ShallowWrapper = shallow(<ButtonAppBar />);

    expect(component).toMatchSnapshot();
  });
});
