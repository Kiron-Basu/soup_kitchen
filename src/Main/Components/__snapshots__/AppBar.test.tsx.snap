// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App AppBar tests Renders AppBar correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(AppBar))
    position="fixed"
  >
    <WithStyles(ForwardRef(Toolbar))>
      <WithStyles(ForwardRef(IconButton))
        aria-label="menu"
        className="makeStyles-menuButton-2"
        color="inherit"
        edge="start"
      >
        <Memo(ForwardRef(KitchenIcon)) />
      </WithStyles(ForwardRef(IconButton))>
      <WithStyles(ForwardRef(Typography))
        className="makeStyles-title-3"
        variant="h6"
      >
        Soup Kitchen
      </WithStyles(ForwardRef(Typography))>
      <Memo(ForwardRef(ShoppingCartIcon)) />
      <WithStyles(ForwardRef(Button))
        color="inherit"
        onClick={[Function]}
      >
        Cart
      </WithStyles(ForwardRef(Button))>
      <ForwardRef(Modal)
        aria-describedby="simple-modal-description"
        aria-labelledby="simple-modal-title"
        className="modal"
        onClose={[Function]}
        open={false}
        tabIndex={-1}
      >
        <WithStyles(ForwardRef(DialogContent))>
          <Connect(OrderSection)
            className="modalBody"
            foodLabels={
              Array [
                "bread",
                "cheese",
                "milk",
                "butter",
                "soup",
              ]
            }
          />
        </WithStyles(ForwardRef(DialogContent))>
      </ForwardRef(Modal)>
    </WithStyles(ForwardRef(Toolbar))>
  </WithStyles(ForwardRef(AppBar))>
</div>
`;
